web/src/components/common/Card.tsx

```diff
@@ -0,0 +1,33 @@
+import { Card as CardWrapper, Typography } from '@equinor/eds-core-react'
+import styled from 'styled-components'
+
+const StyledCard = styled(CardWrapper)`
+  min-width: 250px;
+  background-color: #f7f7f7;
+
+  .Typography__StyledTypography-sc-179guof-0 {
+    color: #007079;
+  }
```

Comment:
This is likely to break. Rather use styled components

Reply:
Disagree. So far, we have no indications that this is likely to break. Nesting rules is even suggested by styled components to reduce the need for classes:

> nesting is a feature intentionally ported from Sass. Used sparingly it's a great way to lighten your code by reducing the need to create explicit classes for every element.
[https://styled-components.com/docs/faqs#can-i-nest-rules](url)

Reply:
Nesting rules are no problem. But refering to css classes by styled-components could be, see; https://stackoverflow.com/questions/57993773/is-it-safe-to-target-styled-components-class-names

So, the className(hash) is calculated based on props and styles. If(when) EDS updates this, our web app will look weird. Not a big deal, there could be other issues with updating packages as well, but a good enough reason that this pattern should be avoided if possible.